neuro_san/http_sidecar/logging/log_context_filter.py

```diff
@@ -9,32 +9,18 @@
 # neuro-san SDK Software in commercial settings.
 #
 # END COPYRIGHT
-"""
-See class comment for details
-"""
+from neuro_san.service.http.logging.log_context_filter import LogContextFilter as ParentLogContextFilter
 
-import contextvars
-import logging
+print("""
+WARNING: The class:
+     neuro_san.http_sidecar.logging.log_context_filter.LogContextFilter
+... has moved to be ...
+    neuro_san.service.http.logging.log_context_filter.LogContextFilter.
+Please update the logging.json file associated with your Neuro SAN agent server accordingly.
+""")
```

Comment:
Not the greatest diff.
The guts of this file moves to neuro_san.service.http.logging.log_context_filter.
What remains is here for backwards compatibility. This print statement above should theoretically get people to move their shit over in their own time.